src/components/BackToPreviousPage/BackToPreviousPage.js

```diff
@@ -0,0 +1,17 @@
+import {MdOutlineArrowBackIosNew} from 'react-icons/md';
+import {useNavigate} from 'react-router-dom';
+import styled from 'styled-components';
+
+export default function BackToPreviousPage() {
+  const navigate = useNavigate();
+  return (
+    <GoBack onClick={() => navigate(-1)}>
```

Comment:
If you want to redirect to as specific page, you could use navigate() to call a path directly, for example to the start page
```suggestion
    <GoBack onClick={() => navigate("/")}>
```

Reply:
Thank you,  good to know. I just need to go to previous page.